includes/media_bynder.admin.inc

```diff
@@ -550,19 +550,13 @@ function media_bynder_search_results($search, $selected_facets, $meta_properties
 
     if ($result) {
         $arr_json = json_decode($result, TRUE);
-
-        //Set total count;
-        if(isset($arr_json['count']['brandId'])){
-            $total_count = 0;
-            foreach($arr_json['count']['brandId'] as $brand_count){
-                $total_count += $brand_count;
-            }
-            $arr_json['totalNrOfResults'] = $total_count;
-        }
+    }
+    if (isset($arr_json['count']['total'])) {
+        $arr_json['totalNrOfResults'] = $arr_json['count']['total'];
     } else {
```

Comment:
this else case seems a bit harsh in for not being able to get the total. Maybe it's supposed to be the else of ```if ($result) {```?

This if itself may need a fallback to calculating it yourself (old code) in the situation count-total is missing, just to be backwards compatible